amcl/src/amcl_node.cpp

```diff
@@ -117,7 +117,7 @@ angle_diff(double a, double b)
     return(d2);
 }
 
-static const std::string scan_topic_ = "scan";
+static const std::string scan_topic_ = "scan_amcl";
```

Comment:
just topic remapping is fine?

Reply:
Yes, in amcl node, this change should be sufficient.